lib/Tmdb/Api/Movies.php

```diff
@@ -137,6 +137,19 @@ public function getSimilar($movie_id, array $parameters = [], array $headers = [
         return $this->get('movie/' . $movie_id . '/similar', $parameters, $headers);
     }
 
+    /**
+     * Get the recommended movies for a specific movie id.
+     *
+     * @param $movie_id
```

Comment:
I would like to maintain coherent camelcasing of variables `$movieId` ( won't repeat it for all the places I come across further on )

Reply:
Just to make sure, you want the `$movie_id` variable to be written as `$movieId`? I'm just confused, because the file, which this code is in (as well as many others), has `$movie_id` as parameter everywhere, and there's no `$movieId` in it